package.json

```diff
@@ -23,9 +23,9 @@
     "build": "react-scripts build",
     "test": "react-scripts test --watchAll=false",
     "eject": "react-scripts eject",
-    "lint": "eslint ./",
-    "lint-fix": "eslint ./ --fix",
-    "format": "prettier --write \"**/*.{js,jsx,json,md}\""
+    "lint": "eslint --fix src/**/*.{js,ts,tsx}",
```

Comment:
We normally wouldn't need to run lint or format as they would be done on the pre-commit, but still good to have it here